fileb0x.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Would it be possible to make this be exclusively a REST API server? I'm trying to move away from bundled resources in the grader too: https://github.com/omegaup/omegaup/commit/c788ec249ddeaed4531c174b0fdaa18428843ab5